File: Bank/app/src/main/java/com/msbm/bank/loginScreen/LoginApi.java
package com.msbm.bank.loginScreen;

import retrofit2.Call;
import retrofit2.http.Field;
import retrofit2.http.FormUrlEncoded;
import retrofit2.http.Headers;
import retrofit2.http.POST;

public interface LoginApi {

    @FormUrlEncoded
    @Headers("Content-Type:application/x-www-form-urlencoded")
    @POST("login")
    Call<LoginResponse> login(@Field("user") String user, @Field("password") String password);
}

File: BankApp/app/src/main/java/com/joaoricardi/bankapp/service/api/LoginApi.kt
package com.joaoricardi.bankapp.service.api

import com.joaoricardi.bankapp.models.login.LoginResponse
import kotlinx.coroutines.Deferred
import retrofit2.http.Field
import retrofit2.http.FormUrlEncoded
import retrofit2.http.Headers
import retrofit2.http.POST

interface LoginApi {

    @FormUrlEncoded
    @Headers( "content-type: application/x-www-form-urlencoded;charset=UTF-8" )
    @POST("login")
    fun login(
        @Field("user")  user: String,
        @Field("password") password: String): Deferred<LoginResponse>
}